e2e/e2etests/e2etests.go

```diff
@@ -448,12 +448,12 @@ var AllE2ETests = []runner.E2ETest{
 		TestSPLWithdraw,
 	),
 	runner.NewE2ETest(
-		TestSPLWithdrawAndCreateReceiverAtaName,
-		"withdraw SPL from ZEVM and create receiver ata",
+		TestSPLWithdrawFailsIfNoReceiverAtaName,
+		"withdraw SPL from ZEVM fails if no receiver ata",
```

Comment:
```suggestion
		"withdraw SPL from ZEVM with no receiver ata",
```
We just need to tell what actions is being performed, not the full context

Reply:
renamed https://github.com/zeta-chain/node/pull/3231/commits/99fdb32958065201558a9627928548ed36e09ec9